make sure that copied case directory has user write permissions after copying;
this may be required if the installation of porefoam was done with --read-only-installdir
author: Kenneth Hoste (HPC-UGent)

diff -ru porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc.orig/src/porefoam1f/script/AllRunImage porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc/src/porefoam1f/script/AllRunImage
--- porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc.orig/src/porefoam1f/script/AllRunImage	2023-12-01 17:27:22.000000000 +0100
+++ porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc/src/porefoam1f/script/AllRunImage	2025-06-10 19:16:38.436900774 +0200
@@ -97,6 +97,8 @@
 	else
 		cp -r $SP_SCRIPTS/base $caseName
 	fi
+	chmod -R u+w $caseName
+
 
 	(cd $caseName/0 && mv p.$direction p)
 	(cd $caseName/0 && rm p.*)
diff -ru porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc.orig/src/porefoam1f/script/AllRunImagePar porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc/src/porefoam1f/script/AllRunImagePar
--- porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc.orig/src/porefoam1f/script/AllRunImagePar	2023-12-01 17:27:22.000000000 +0100
+++ porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc/src/porefoam1f/script/AllRunImagePar	2025-06-10 19:16:50.494243064 +0200
@@ -108,6 +108,7 @@
 	else
 		cp -r $SP_SCRIPTS/base $caseName
 	fi
+	chmod -R u+w $caseName
 
 	(cd $caseName/0 && mv p.$direction p)
 	(cd $caseName/0 && rm p.*)
diff -ru porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc.orig/src/porefoam1f/script/AllRunImageParDistributed porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc/src/porefoam1f/script/AllRunImageParDistributed
--- porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc.orig/src/porefoam1f/script/AllRunImageParDistributed	2023-12-01 17:27:22.000000000 +0100
+++ porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc/src/porefoam1f/script/AllRunImageParDistributed	2025-06-10 19:16:50.499243205 +0200
@@ -98,6 +98,7 @@
 	else
 		cp -r $SP_SCRIPTS/base $caseName
 	fi
+	chmod -R u+w $caseName
 
 	(cd $caseName/0 && mv p.$direction p)
 	(cd $caseName/0 && rm p.*)
diff -ru porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc.orig/src/porefoam1f/script/AllRunImageParSaveMemory porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc/src/porefoam1f/script/AllRunImageParSaveMemory
--- porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc.orig/src/porefoam1f/script/AllRunImageParSaveMemory	2023-12-01 17:27:22.000000000 +0100
+++ porefoam-bce1ff40d6d69bfcbe569da09959b97df0ea12fc/src/porefoam1f/script/AllRunImageParSaveMemory	2025-06-10 19:16:50.504243347 +0200
@@ -114,6 +114,7 @@
 
 
 	cp -r $SP_SCRIPTS/base $caseName
+	chmod -R u+w $caseName
 
 
 	(cd $caseName/0 && mv p.$direction p)
